src/main/java/utils/ApiHelper.java

```diff
@@ -0,0 +1,95 @@
+package utils;
+
+import api.ProductCode;
+import api.ProductQuantity;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.restassured.RestAssured;
+import io.restassured.module.jsv.JsonSchemaValidator;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import static org.hamcrest.Matchers.equalTo;
+
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ApiHelper {
```

Comment:
On a real life projects you will have a separate class for each of the resources, so you separate the responsibility. It's not the ApiHelper work to know about the Cart/User/Checkout resources and the respective details